### PR TITLE
[WIP] windows build

### PR DESCRIPTION
**Pull request by BubblyOrca**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by cbrueffer** [comment]
Hi @BubblyOrca, just adding a new maintainer without explanation or actual content changes doesn't tell us much. Please submit the actual changes you'd like to make via PR and if you're still interested, add yourself as maintainer as part of that.

That said, this recipe could certainly use a cleanup.

**Comment by BubblyOrca** [comment]
Hi,

I’m attempting to do a windows fix. However, I just recently opened this PR
and I immediately ran into some issues with the linter. After I address that
you’ll see the cleanup action items appear.

I can change the PR message to “windows fix” so it’s more clear for you.

**Comment by cbrueffer** [comment]
Great, that's very useful! We just had a few PRs in other feedstocks that added a maintainer and nothing else, so I thought I'd chime in here since we're unlikely to merge those. And yes, if that's what you'll do in this PR, please change the title.

**Comment by cbrueffer** [comment]
As a starting point I'd suggest re-generating the recipe using `https://github.com/bgruening/conda_r_skeleton_helper`, that'll clean the recipe up and provide a good base for anything further.

**Comment by BubblyOrca** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,9 +13,8 @@ source:
   sha256: 9d99d3d0a4b7770c3c3a6de44206811272d78ab94481713a8c369f7d6ae7b80f
 
 build:
-  number: 1005
+  number: 0
```
Comment (dbast) [inline recipe/meta.yaml:16]: This can only be set to 0, if the version is updated.

**Comment by dbast** [comment]
@BubblyOrca have a look at https://github.com/conda-forge/r-gensa-feedstock/pull/9 .. used https://github.com/bgruening/conda_r_skeleton_helper to regenerate the recipe + another automatic rerender commit. That added all the windows build requirements and everything is green there.

**Comment by dbast** [comment]
Done via https://github.com/conda-forge/r-gensa-feedstock/pull/9

**Comment by BubblyOrca** [comment]
> Done via https://github.com/conda-forge/r-gensa-feedstock/pull/9

@dbast Hi! I wanted to verify if the windows build has been updated and merge for r-gensa since my PR is closed?

I needed to incorporate the windows build for r-gensa because it's a dependency for r-eulerr which is the next feedstock I'm moving on to.

**Comment by dbast** [comment]
Please have a look at https://github.com/conda-forge/r-gensa-feedstock/pull/9 and https://anaconda.org/conda-forge/r-gensa ... that shows you everything.

**Pull request closed without merging**